tex/interpretable_ml.tex

```diff
@@ -54,6 +54,8 @@
 
 \title{\large \bfseries Interpretable Machine Learning}
 
+\author{Wen Phan, Lauren DiPerna, Patrick Hall\\[3ex]}
+
```

Comment:
Can you please add me as an author to the end of this? I did edits locally before the initial push with no authors listed.

Reply:
I will add you as author and sections for PDP and ICE.

Reply:
Sections for PDP and ICE? Those should be there in this text.

Reply:
Yep - they are there now - merging. 

Addresses #14